orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -128,7 +128,7 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
     Set<String> inferredRegions = new HashSet<>()
 
     deployRegions.forEach {
-      if (!config.regions.contains(it)) {
+      if (!config.regions.contains(it) && cloudProvider == 'aws') {
```

Comment:
prob would be more readable if it were
```
if (cloudProvider == 'aws') {
      deployRegions.forEach {
            if (!config.regions.contains(it) && cloudProvider == 'aws') {
....
}
```
or, maybe just have the conditional on line 127, that way, we don't even bother looking at child stages unless it's `aws`?

Reply:
Agree, no point iterating `deployRegions` if !aws, will update